pyotgw/pyotgw.py

```diff
@@ -461,9 +494,9 @@ def get_gpio_mode(self, gpio_id):
         """
         if not self._connected:
             return
-        return self._protocol.status.get("OTGW_GPIO_{}".format(gpio_id))
+        return self._protocol.status.get(f"v.OTGW_GPIO_{gpio_id}")
```

Comment:
Same

Reply:
Fixed in https://github.com/mvn23/pyotgw/pull/12/commits/f6ed560b474bbd63057d40757dc13a2988a73c72.